src/envelope.rs

```diff
@@ -57,6 +58,9 @@ struct Instrumentation {
     _waiting_for_actor: tracing::Span,
 }
 
+#[derive(Clone)]
+pub struct HandlerSpan(#[cfg(feature = "instrumentation")] pub tracing::Span);
```

Comment:
Do we need this encapsulation with the new `TickFuture`?

Reply:
It's still used elsewhere in `envelope` iirc